phpstan.neon

```diff
@@ -17,6 +17,7 @@ parameters:
         - app/Http/ViewHelpers/Company/Group/GroupMeetingsViewHelper.php
         - app/Http/ViewHelpers/Company/Group/GroupShowViewHelper.php
         - app/Http/ViewHelpers/Company/Group/GroupViewHelper.php
+        - app/Http/ViewHelpers/Adminland/AdminSoftwareViewHelper.php
```

Comment:
why?

Reply:
Because I'm lazy and didn't want to fix the code, but I've fixed it, so I can remove the line.